client/src/components/Ticketing/checkout/CompleteOrderForm.tsx

```diff
@@ -111,9 +111,9 @@ export default function CompleteOrderForm(
       <div className='min-w-414 sm:w-full h-full'>
         <Form
           onSubmit={handleSubmit}
-          initialValues={{'opt-in': false, 'seating-accommodation': false}}
+          initialValues={{'opt-in': false}}
           render={({handleSubmit}) => (
-            <form onSubmit={handleSubmit} noValidate className='w-full h-full bg-zinc-200 p-4 rounded-xl flex flex-col  justify-between'>
+            <form onSubmit={handleSubmit} noValidate className='w-full h-full bg-zinc-200 p-2 md:p-4 rounded-xl flex flex-col  justify-between'>
               <div className='flex flex-col w-auto'>
                 <div className='grid gap-5 md:grid-cols-2'>
```

Comment:
What do you think about changing this grid break point to md(768px) rather than the custom md(960px)?

Reply:
When i changed md to 768px, it doesn't look right.
md-768px
![mobile04_after_md768](https://github.com/WonderTix/WonderTix/assets/97830813/0f0317f6-8437-41e5-a51e-ff5b43a9b338)

md-960px
![mobile05_md960](https://github.com/WonderTix/WonderTix/assets/97830813/c01988c6-2167-47cd-848f-12a3c0e7ccd8)

Reply:
i think thats a good idea.  having more options is always better